libp2p/protocols/pubsub/gossipsub/behavior.nim

```diff
@@ -25,6 +25,7 @@ declareGauge(libp2p_gossipsub_no_peers_topics, "number of topics in mesh with no
 declareGauge(libp2p_gossipsub_low_peers_topics, "number of topics in mesh with at least one but below dlow peers")
 declareGauge(libp2p_gossipsub_healthy_peers_topics, "number of topics in mesh with at least dlow peers (but below dhigh)")
 declareCounter(libp2p_gossipsub_above_dhigh_condition, "number of above dhigh pruning branches ran", labels = ["topic"])
+declareSummary(libp2p_gossipsub_mcache_hit, "ratio of successful IWANT message cache lookups", labels = ["topic"])
```

Comment:
not set. This would make sense as a per-topic meric, but I see IHAVE and other messages have a topicID field, but IWANT does not. 
If we still have the message in mcache, we could get the topic ID. But if it is missing, we just simply don't know, I'm afraid.
Consequence: if you agree, lets just remove labels here

Reply:
if you don't set declared labels, metrics will crash at runtime

Reply:
Yeah, let's remove it